### PR TITLE
Add Flux BoF video to resources

### PR DESCRIPTION
**Pull request by kingdonb**
I was looking for this one - let's post it first.

If there are any others mention them here and I'll add them before it merges. I think if anyone publishes a Flux-centric KubeCon Recap post, we could also cross-link it on the Flux website using our new remote resources feature

### CODE REVIEW

**Comment by kingdonb** [comment]
For now, the list of Flux talks at KubeCon is still up and available here:
https://fluxcd.io/kubecon/

(It is maintained at https://github.com/fluxcd/community/commit/69a1e51f0e48f9d8a9405cfd681062860752a71b and will eventually move from there, for next KubeCon)

If anyone wants to comb through the KubeCon already published talks and add their talk here, send them my way. I have only  added a few, I think that at least talks from Wednesday are up. Checking for formatting and that the links are correct before I try to add any more.

**Comment by kingdonb** [comment]
More links I did not include yet:

- [ ] [How We Are Moving from GitOps to Kubernetes Resource Model in 5G Core](https://www.youtube.com/watch?v=crmTnB6Zwt8)
- [ ] [Lightning Talk: My Database Runs on Kubernetes. What's Next? Data Platforms! - Robert Hodges](https://www.youtube.com/watch?v=nRIkZQOhOWo)
- [ ] [Panel: Argo Vs Flagger: Progressive Delivery... - Scott Rigby, Anastasiia Gubska & Kostis Kapelonis](https://www.youtube.com/watch?v=W-xTeBTQ2kI)

I didn't watch them yet either, (I haven't actually seen any of these - time to take a deep dive in)

**Comment by kingdonb** [comment]
Deploy preview at:

https://deploy-preview-1884--fluxcd.netlify.app/resources/

File: data/resources.yaml [hunk 1]
```diff
@@ -13,6 +13,18 @@ resources:
   # Note: To generate "thumbnail_url", visit https://embed.ly/docs/explore/extract
   # and enter the link to the resource. Once you do that, select any thumbnail url of choice to use.
 
+  - youtube: LTwJIMWovO4
+    title: "KubeCon EU 2024: Flux: Secure and Scalable GitOps | Project Lightning Talk"
+    date: "2024-03-26"
+    type: video
+  - youtube: rJ6vebxS0aI
+    title: "KubeCon EU 2024: Tackling Configuration Management at Scale with Flux, CUE and OCI at... Alec Hothan & Stefan Prodan"
```
Comment (stefanprodan) [inline data/resources.yaml:21]:
```suggestion
    title: "KubeCon EU 2024: Tackling Configuration Management at Scale with Flux, CUE and OCI at Cisco | Alec Hothan & Stefan Prodan"
```

**Review by stefanprodan — COMMENTED** [review]
There is also https://www.youtube.com/watch?v=JFLNFJT59DY

File: data/resources.yaml (hunk 1 above)
Comment (kingdonb) [reply data/resources.yaml]: Fixed in 71ab400

**Comment by kingdonb** [comment]
I'm looking through the talks I did not list, I think I won't include:

* [Lightning Talk: My Database Runs on Kubernetes. What's Next? Data Platforms! - Robert Hodges](https://www.youtube.com/watch?v=nRIkZQOhOWo)

It's a great talk, I like the message, but while it talks about ArgoCD and "Not ArgoCD" the presenter never gets around to mentioning Flux, (so I'm not sure why and how it's gotten on our list!)

The Swisscom talk comes out and mentions they do use Flux, so I'll add that one, and the Flagger/Argo Rollouts panel

Then merge this 👍

**Pull request merged**